src/handlers/motions/motionVoteRevealed/helpers.ts

```diff
@@ -0,0 +1,42 @@
+import { BigNumber } from 'ethers';
+
+import { MotionMessage, MotionEvents } from '~types';
+
+interface Props {
```

Comment:
Props sound a bit odd in a non-React context. How about `Options`, `Params`?

Reply:
Params sounds good